### DISCUSSION TITLE
IE 11 support

### DISCUSSION THREAD

**Discussion by narensisodiya** [xyflow/xyflow#1063] [Q&A]
Hello, 

Thanks for building this great library. 

Is there any documentation for supported browsers? does it support for IE 11?

Thanks,
N a r e n

**Comment by moklick** [comment]
IE 11 is not supported but it's possible by adding polyfills. If some one could tell me which polyfills are needed I will add that inforamtion to the docs.

**Reply by narensisodiya** [reply]
took a chance to trying out on IE 11 and it failed with loading D3-dispatch in IE11 because of arrow function..
<img width="713" alt="d3_dispatch_syntax_error" src="https://user-images.githubusercontent.com/491563/113765976-c9a07700-96d1-11eb-8011-06a283942079.png">

and going further more.. i have found that version 2.0 has dropped support of IE11 and recommending using old version or transpiler 
https://github.com/d3/d3-dispatch/releases/tag/v2.0.0

please suggest if there is a way to handle this?

**Reply by nurikk** [reply]
Even Microsoft does't support IE11 anymore

**Reply by moklick** [reply]
@narensisodiya if that's the problem you need to transpile the unsupported modules with babel to es5.
